pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>jv-fruit-shop</artifactId>
+    <artifactId>jv-fruitTransaction-shop</artifactId>
```

Comment:
Do not change the pom.xml file. Rollback all changes before re-request new review.